README.md

```diff
@@ -1 +1,18 @@
 # wrf-docker
+
+This build system creates a docker container for WRF / WPS.
+
+Main page: https://github.com/wrf-model/WRF
+
+## Requirements
+
+* Docker (version 18.09 or later)
+
+## Usage
+
+Begin with building the base container, this needs to be named `wrf_intermediate`:
+* `DOCKER_BUILDKIT=1 docker build . -t 'wrf_intermediate' -f Dockerfile-base`
```

Comment:
For building for release the full tag would be needed e.g. oliverwoolland/wrf_intermediate - not sure if this should be in this doc or not

Reply:
When we have a permanent home for the docker images then we should add that - but for now I don't think we need to (as we're not expecting others to build these directly).